pkg/storage/system_intake_test.go

```diff
@@ -642,6 +642,78 @@ func (s StoreTestSuite) TestFetchSystemIntakesByFilter() {
 			s.True(found, "did not receive expected intake", id)
 		}
 	})
+
+	s.Run("collects most recent admin notes", func() {
+		ctx := context.Background()
+
+		var intakeIDs []uuid.UUID
+
+		for i := 0; i < 5; i++ {
+			intake := testhelpers.NewSystemIntake()
+			result, err := s.store.CreateSystemIntake(ctx, &intake)
+			s.NoError(err)
+			intakeIDs = append(intakeIDs, result.ID)
+		}
+
+		intakeWithNoCommentsID := intakeIDs[0]
+		intakeWithOneCommentID := intakeIDs[1]
+		intakeWithManyCommentsID := intakeIDs[2]
+
+		_, err := s.store.CreateNote(ctx, &models.Note{
+			SystemIntakeID: intakeWithOneCommentID,
+			Content:        null.StringFrom("the only comment"),
+			CreatedAt:      mustParseTime("2021-01-01"),
+			AuthorEUAID:    "ABCD",
+		})
+		s.NoError(err)
+
+		_, err = s.store.CreateNote(ctx, &models.Note{
+			SystemIntakeID: intakeWithManyCommentsID,
+			Content:        null.StringFrom("the first comment"),
+			CreatedAt:      mustParseTime("2021-01-01"),
+			AuthorEUAID:    "ABCD",
+		})
+		s.NoError(err)
+		_, err = s.store.CreateNote(ctx, &models.Note{
+			SystemIntakeID: intakeWithManyCommentsID,
+			Content:        null.StringFrom("the third comment"),
+			CreatedAt:      mustParseTime("2021-01-03"),
+			AuthorEUAID:    "ABCD",
+		})
+		s.NoError(err)
+		_, err = s.store.CreateNote(ctx, &models.Note{
```

Comment:
This note is intentionally created with an earlier creation date but after the other ones to verify that we're applying the correct order to the notes in the database.